### PR TITLE
Delay promise for revoke

### PR DESCRIPTION
**Pull request by avishalom**
Adding a 1 second timeout in the integration tests to make sure revoke happens 1 second after minting. also comment

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "@firebase/auth": {
```
Comment (hiranya911) [inline package-lock.json]: It is weird to have changes in the lock file without corresponding changes in `package.json`. Please revert.

File: src/auth/auth-api-request.ts [hunk 2]
```diff
@@ -536,6 +536,9 @@ export class FirebaseAuthRequestHandler {
    * In addition to revoking all refresh tokens for a user, all ID tokens issued
    * before revocation will also be revoked on the Auth backend. Any request with an
    * ID token generated before revocation will be rejected with a token expired error.
+   * Note that due to the fact that the timestamp is stored in seconds, any tokens minted in
+   * the same second as the revokation will still be valid. If there is a chance that a token
```
Comment (hiranya911) [inline src/auth/auth-api-request.ts]: typo: revokation --> revocation

File: test/integration/auth.spec.ts [hunk 3]
```diff
@@ -167,7 +167,9 @@ describe('admin.auth', () => {
       })
       .then((decodedIdToken) => {
         // Verification should succeed. Revoke that user's session.
-        return admin.auth().revokeRefreshTokens(decodedIdToken.sub);
+        return new Promise(resolve => setTimeout(() => resolve(
```
Comment (hiranya911) [inline test/integration/auth.spec.ts]: (resolve) =>

File: src/auth/auth-api-request.ts [hunk 4]
```diff
@@ -536,6 +536,9 @@ export class FirebaseAuthRequestHandler {
    * In addition to revoking all refresh tokens for a user, all ID tokens issued
    * before revocation will also be revoked on the Auth backend. Any request with an
    * ID token generated before revocation will be rejected with a token expired error.
+   * Note that due to the fact that the timestamp is stored in seconds, any tokens minted in
+   * the same second as the revocation will still be valid. If there is a chance that a token
+   *  was minted in the last second, delay for 1 second before revoking.
```
Comment (bojeil-google) [inline src/auth/auth-api-request.ts]: Extra space before `was minted`

**Pull request merged**